src/storybook/stand-alone-documentaion/accessibility/assets/index.js

```diff
@@ -0,0 +1,27 @@
+import colorContrastImage from "./color-contrast_image.png";
+import colorContrastText from "./color-contrast_text.png";
+import hierarchyOrder from "./hierarchy_order.png";
+import imageryDecorative from "./imagery_decorative.png";
+import imageryDescriptive from "./imagery_descriptive.png";
+import imageryAlt from "./imagery_alt.png";
```

Comment:
I think that using code as picture will not be very helpful for developers. we will need to change it really soon.

Reply:
i agree , let's schedule a separate effort for this.